eng/publishPackages/publish.proj

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.DotNet.Arcade.Sdk">
+  <PropertyGroup>
+    <ArtifactsPkgDir>$(RepoRoot)artifacts/packages/</ArtifactsPkgDir>
+    <ManifestDir>$(RepoRoot)artifacts/asset-manifests/</ManifestDir>
+    <TarballDir>$(RepoRoot)artifacts/tarball/</TarballDir>
+    <RestoreSources>
+      $(RestoreSources);
+      https://dotnetfeed.blob.core.windows.net/dotnet-tools-internal/index.json;
```

Comment:
This might need to be a nuget.config to allow for internal servicing.

Reply:
Do you know if it will fall back to the root NuGet.config or do we need a separate one for this?